src/ui/ons/partials/ONSError.svelte

```diff
@@ -0,0 +1,18 @@
+<script>
+  export let message, id;
```

Comment:
I actually prefer `message` than `text` but I think we should stick to [whatever DS uses in their Nunjuck examples](https://ons-design-system.netlify.app/components/input/#error-messages), in this case it's `text` for the error message:
```json
"error": {
  "id": "email-error",
  "text": "Enter an email address in a valid format, for example, name@example.com",
}
```

Reply:
As per your suggestion I've renamed a few props of the <ONSTextField /> and <ONSError /> components to follow the DS examples : 

1.  <ONSError /> : message(old) >> errorText(new)
2. <ONSTextField />: inputLabel(old) >> labelText (new) ; showPlaceholder(old) >> accessiblePlaceholder(new)